torchtune/models/gemma/_model_utils.py

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+from torchtune.modules import TransformerDecoder
+
+
+def tie_weight(model: TransformerDecoder) -> None:
```

Comment:
Where is this used?

Reply:
This was used in a previous version and now I removed it.